configure.ac

```diff
@@ -161,7 +161,7 @@ AC_ARG_WITH([scl],
                             [Build against Software Collections RPM])],
             [scl_enabled_build=:])
 
-SYSTEMD_EXECSTART='${bindir}/postmaster -D $${PGDATA}'
+SYSTEMD_EXECSTART='${bindir}/postgres -D $${PGDATA}'
```

Comment:
This seems like the only non-documentation change, right?  `/usr/bin/postgres` is available even in PostgreSQL 9.2 (RHEL 7).  LGTM